Classes/common/CDTFetchChanges.h

```diff
@@ -67,6 +67,28 @@
  */
 @property (nonatomic, copy) NSString *startSequenceValue;
 
+/**
+ Use this property to limit the total number of results you want to get.
+ 
+ By default, its value 0 which means that it will returns as many as possible.
+ 
+ Set it before executing the operation or submitting it to a queue.
+ */
+@property(nonatomic, assign) NSUInteger resultsLimit;
```

Comment:
It should be documented that if the resultsLimit is above the default defined in the implementation it will be ignored and the default used.